test/test_other.py

```diff
@@ -3297,13 +3297,19 @@ def test_embind_tsgen_memory64(self):
                      self.get_emcc_args())
     self.assertFileContents(test_file('other/embind_tsgen_memory64.d.ts'), read_file('embind_tsgen_memory64.d.ts'))
 
-  def test_embind_tsgen_exceptions(self):
+  @parameterized({
+    '': [0],
+    'wasm_exnref': [1]
+  })
+  def test_embind_tsgen_exceptions(self, wasm_exnref):
+    self.set_setting('WASM_EXNREF', wasm_exnref)
```

Comment:
Presumably this should have:

```
if wasm_exnref: 
  self.requires_wasm_enxref()
```

Reply:
This only runs compilation and does not run it. Note that there was no `self.require_wasm_eh()` before.